data/scripts/dockerfiles/database_setup.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE IF NOT EXISTS drug_label_explorer;
+
+USE drug_label_explorer;
+
+CREATE TABLE IF NOT EXISTS drug_labels (
```

Comment:
@llandau I wonder how similar of a schema you came to with EU data

Reply:
see comment below

Reply:
Thanks for the quick feedback! Some more context
1. Docker + DB is to test locally. Whether or not the script is used for production deployment we can decide, but this allows the building of Maria DB to confirm that the Schema defined in create table is bug free.
2. Yes, I did not push that up as it is still WIP until the schema is finalized.
3. No I created the schema in the `database_setup.sql`file. We can add a Django Model once the server is up and running.
4. I agree. That is why I wrote all the scripts needed to build a database and insert a record. Are you doing this work?
5. The CSV is just for testing.
6. How? The point of this is to define the schema, and I do not think there is any clear indication for how we should break up the label field, so this is what I am suggesting.
7. Ok. I will move forward assuming this is the schema we will use and adjust later if needed.
8. Saturday midday I am free, Saturday evening I am not. I am also free Sunday morning/midday. We can discuss tomorrow.